pkg/api/jokes.go

```diff
@@ -35,7 +35,10 @@ func (h Handler) addJoke(w http.ResponseWriter, r *http.Request) {
 	title := r.FormValue("title")
 	body := r.FormValue("body")
 
-	h.storage.AddJoke(title, body)
+	err := h.storage.AddJoke(title, body)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
1) I believe you need return here
2) Why do not write error to the response body ?